ee/webhooks/cmd/root.go

```diff
@@ -26,7 +26,10 @@ func NewRootCommand() *cobra.Command {
 	}
 
 	otlptraces.InitOTLPTracesFlags(root.PersistentFlags())
-	publish.InitCLIFlags(root)
+	publish.InitCLIFlags(root, func(cd *publish.ConfigDefault) {
```

Comment:
Since all apps obey to the same rules with your modification, perhaps, you could set -1 as default.